site/en/docs/extensions/reference/index.md

```diff
@@ -1,13 +1,11 @@
 ---
-title: 'API Reference'
+title: 'API reference'
 description: 'The complete reference to all APIs made available to Chrome Extensions. This includes APIs for the deprecated Chrome Apps platform as well as APIs still in beta and dev.'
 layout: 'layouts/reference-landing.njk'
 ---
 
-Chrome provides extensions with many special-purpose APIs like `chrome.runtime` and `chrome.alarms`.
+Chrome provides extensions with many special-purpose interfaces such as `chrome.runtime` and `chrome.alarms`.
```

Comment:
@jpmedley Can you get @dotproto's input on using the term "interfaces" here? I don't see a problem, but I wanted double check before merging.

Reply:
'Interfaces' is gone. After discussing this with Simeon, the new text reflects how we think APIs should be described. There's an email about this in your inbox.